devicetypes/smartenit/smartenit-zigbee-metering-outlet.src/smartenit-zigbee-metering-outlet.groovy

```diff
@@ -0,0 +1,138 @@
+/*
+ *	Copyright 2016 SmartThings
+ *
+ *	Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ *	use this file except in compliance with the License. You may obtain a copy
+ *	of the License at:
+ *
+ *		http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *	Unless required by applicable law or agreed to in writing, software
+ *	distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ *	WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ *	License for the specific language governing permissions and limitations
+ *	under the License.
+ */
+ import groovy.transform.Field
+ 
+ @Field final MeteringCurrentSummation = 0x0000
+ @Field final MeteringInstantDemand = 0x0400
+ @Field final Current = 0x0508
+ @Field final Voltage = 0x0505
+ @Field final EnergyDivisor = 100000
+ @Field final CurrentDivisor = 1000
+ @Field final SmartenitMfrCode = 0x1075
+ @Field final ElectricalMeasurement = 0x0b04
+ @Field final ACVoltageMultiplier = 0x0600
+ @Field final ACVoltageDivisor = 0x0601
+ @Field final ACCurrentMultiplier = 0x0602
+ @Field final ACCurrentDivisor = 0x0603
+ @Field final ActivePower = 0x050b
+ @Field final PowerMultiplier = 0x0604
+ @Field final PowerDivisor = 0x0605
+ @Field final ReportingResponse = 0x07
+
+metadata {
+	// Automatically generated. Make future change here.
+	definition(name: "Smartenit Zigbee Metering Outlet", namespace: "Smartenit", author: "Luis Contreras", mnmn: "SmartThingsCommunity", vid: "9f4df74b-f0d4-3515-9384-f5297ee3b11c", ocfDeviceType: "oic.d.smartplug", minHubCoreVersion: '000.017.0012') {
```

Comment:
Double-checking:
Did You also check if Your device is working with this DTH ?
https://github.com/SmartThingsCommunity/SmartThingsPublic/blob/master/devicetypes/smartthings/zigbee-metering-plug.src/zigbee-metering-plug.groovy

Reply:
I tried using this DTH but ever since the SmartThings Edge release, I've been having trouble making modifications to DTHs.

Reply:
To be clear: you were copying the source and then self-publishing as your own DTH with modifications, right?

Either way, since you're using your custom capability with the "current" attribute, this would need to be a new DTH.